on-off-board/onboarding.md

```diff
@@ -4,77 +4,96 @@ Welcome to NYPL Digital!
-  - Contains links to development standards and guidelines at NYPL.
-- Review READMEs (or Wikis) in relevant repos
-  - Generally, each repo will contain more app-specific information.
+We hope that you'll have a good first at NYPL Digital! Generally 4 things should happen your first day:
 
-## 2. Set up accounts
+1. You'll meet your team and have a lunch or other meet-and-greet
+2. You'll get your laptop and have some time to set it up
+3. You'll be given an onboarding packet that lays out your first day, week, month and quarter
+4. You'll start meeting with your team members and collaborators to start planning your work
 
-### Commonly used accounts
+After that your intro will vary depending on your team and role, but it should focus at least in these areas
 
-A developer will generally need accounts for the following services:
+## 1. Learning
 
-- Slack
-  - [nypl.slack.com](https://nypl.slack.com/)
-- GitHub (Can use a personal account or make a new NYPL specific GitHub account)
+There is a lot to learn in Digital, and there is no way to pick everything up in your first month, let alone your first day! So here are some tips for starting the process of learning the whats, wheres, hows and whys of our department.
+
+- Review the [Contents](../README.md#contents) of this repo.
+- Review READMEs (or Wikis) of the repositories your team maintains
+  - Your manager and onboarding buddy should help give you context for them
+  - A good approach is to spin up local dev environments for any repositories you'll be contributing to.
+- Set up meetings with other engineers on your team to get a sense for the status of the team's work and where you can help contribute!
+- Review the documents and initial tasks outlined in the specific onboarding documents provided to you on your first day (Which is probably how you ended up here!)
+- Ask questions. If your manager did not ask you to do this already: start a document with questions and things that were unclear!
+  - This document should be an important part of your early 1:1s with your manager
+  - We want to use these questions to improve our documentation and processes! People with fresh eyes often see things that we miss
+
+## 2. Account Setup
+
+### Password Management
+
+We __strongly__ encourage team members to use a password manager for their credentials. There will be many passwords you need to track and the dangers of password reuse are real. We recommend one of the following password management tools:
+
+- [Keeper](https://keepersecurity.com)
+  - This is NYPL's approved password manager
+  - To create a keeper account attempt to sign in with your `@nypl.org` email address, you should be prompted to request Admin approval.
+- [MacPass](https://macpassapp.org/)
+  - This is a local, open source, password management solution
+  - It is not officially supported, but provides an easy place to track passwords
 
-  - NYPL GitHub organization:
+### Commonly used accounts
```

Comment:
The off-boarding document cleverly links to this document rather than restate all of the accounts. We may want to update [that link](https://github.com/NYPL/engineering-general/blob/ce7d7466b1ee992f60823db2e565ac5ce3d314ec/on-off-board/offboarding.md?plain=1#L7) from `[accounts](onboarding.md#2-set-up-accounts)` to `[accounts](onboarding.md#commonly-used-accounts)` so the deep link resolves after merging this.

Reply:
Good catch! I'll update that